### ISSUE TITLE
error:  no reads in first split: bad BAM file or tiny split size?

### ISSUE THREAD

**Issue by astewart-twist** [bigdatagenomics/adam#896]
I keep receiving the following error while trying to convert a bam file into adam format.  

```
java.io.IOException: 'file:/my.bam': no reads in first split: bad BAM file or tiny split size?
```

Additional info:

 * I'm able to run the small.sam test set
 * I've regenerated my bam file multiple times and checked for any kind of corruption, etc
 * I've repeated this experiment on both a local environment (spark in single machine mode), and in EMR.
 * spark-1.5.2-bin-hadoop2.6
 * adam-distribution_2.10-0.18.2
 * scala version 2.10.6

Full stack trace: https://gist.github.com/astewart-twist/ceb36e492f96fe2a2500

**Comment by heuermh** [comment]
@fnothaft @jpdna I was never able to reproduce this, might it be related to #962?

**Comment by fnothaft** [comment]
Nah, #962 would be output only, not input.

**Comment by heuermh** [comment]
Ah right, sorry, wasn't looking closely enough

**Comment by heuermh** [comment]
Fixed by https://github.com/HadoopGenomics/Hadoop-BAM/pull/80 upstream, will have to wait for an update of Hadoop-BAM dependency version to 7.4.1 (or later as appropriate) to pick it up.

**Comment by heuermh** [comment]
See #989 which uses Hadoop-BAM version 7.4.1-SNAPSHOT to pick up the fix for this issue.  Currently blocked on Jenkins environment changes.

**Comment by fnothaft** [comment]
@shaneknapp should be back online tomorrow. I looked and alas, I do not have permissions to make the @AmplabJenkins changes.

**Comment by heuermh** [comment]
May be fixed by #989, please retest if possible.

**Comment by fnothaft** [comment]
Closing as fixed by #989.
